### PR TITLE
Ammo Re-Calibration; Weapon Caliber Rework

### PR DESCRIPTION
**Pull request by Stutternov**
## About The Pull Request
<details>
<summary>
	Simply put, this PR adjusts our hodge-podge of metric and imperial/caliber system into just metric measurements. All imperial measurements have been transfered or adjusted to either: A. Where they make sense (ex .50 short / 'action express' style Kurtz -> 11mm (Equal to .45 ACP or .44 Magnum IRL)) or B. Simply relabeled to what the caliber equals to in metric measurements (.35 inch = 9mm). Furthermore I've kept 'Carbine', 'Rifle', 'Heavy Rifle', and other indicators to make it easy to understand at a glance for players what mag/caliber goes to what sort of gun. This PR is mostly flavor/lore changes but effectively redoes all ammo types to the following:
	
	.35 -> 9mm Standard
	.40 -> 10mm Magnum 
	.50 -> 12mm Heavy Pistol
	.70 -> 19mm Explosive
	10x24mm -> Unchanged
	.223 Laser -> 5.56mm Laser
	.257 -> 6.5x39mm Carbine
	7.5 -> 7.62x39mm Rifle (It was 7.62 before. Not sure why it was changed to 7.5mm)
	.408 -> 8x70mm Heavy Rifle
	.60-06 -> 14.5×114mm Anti-Material
	30mm -> 17mm (proper musket-ball sized)
	40mm -> Unchanged
	.22 Flare -> Flare Shell (Just removed .22. Doesn't need a caliber.)
	20mm Shotgun -> Unchanged
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:
tweak: Redoes all ammo calibers to be standardized to Metric.
/:cl:

### CODE REVIEW

**Comment by JohnWilsonDeez** [comment]
Facts so true my brother

**Comment by cdb-is-not-good** [comment]
Sad 😭😭😭😭 I'll miss you .408

**Comment by TheSleepyCatmom** [comment]
Just for the note, I approve it from my side of things if that matters, just to put it out here officially -- whenever it gets working. Though I do think the "x##" should go still, and just be replaced with a letter or something if it needs to be stated, since the cartridge length isn't really all that important here, since it's not like we have multiple types of the same bullet with different cartridges. But that's just me.

Also if you wanted to swap 11mm back to 12mm that's fine too.

**Comment by ThePainkiller** [comment]
Suggestion:

.50 => 12.7 mm (11m is too small a bore to fit)

**Comment by Stutternov** [comment]
Putting this on hold for a day to get a few lore considerations out of the way regarding a possible change to .408's casing diameter. (We have like 3 10mms rn, need to figure out something a bit better.)

File: code/modules/projectiles/guns/projectile/pistol/handmade.dm [hunk 1]
```diff
@@ -48,7 +48,7 @@
 /obj/item/gun/projectile/handmade_pistol/anti_material
 	name = "handmade man-opener"
 	desc = "An unreliable hand-crafted pistol liable to blow up in your hands, while it has a chance to jam with every shot its easy and cheap to make with customizable calibers. \
-	Reloading, clearing jams, and opening or closing the chamber is done with a screwdriver. This one chambers up to one .60-06 anti material ammo. What mad man made this? It's jam chance and recoil \
+	Reloading, clearing jams, and opening or closing the chamber is done with a screwdriver. This one chambers up to one 14.5x114mm anti material ammo. What mad man made this? It's jam chance and recoil \
```
Comment (ThePainkiller) [inline code/modules/projectiles/guns/projectile/pistol/handmade.dm:51]: Please fix the typo while at it, should be "its"

**Pull request merged**